manifests/r/rcmaehl/MSEdgeRedirect/0.7.5.3/rcmaehl.MSEdgeRedirect.locale.en-US.yaml

```
@@ -1,16 +1,13 @@
-# Created with WinGet Releaser v2 using Komac v1.11.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
-
 PackageIdentifier: rcmaehl.MSEdgeRedirect
 PackageVersion: 0.7.5.3
 PackageLocale: en-US
 Publisher: Robert Maehl Software
 PublisherUrl: https://msedgeredirect.com
-PublisherSupportUrl: https://github.com/rcmaehl/MSEdgeRedirect/issues
+PublisherSupportUrl: https://paypal.me/rhsky

```

```suggestion
PublisherSupportUrl: https://github.com/rcmaehl/MSEdgeRedirect/issues
```
This is not the URL to donate to the publisher, it is where to go to get help with issues